README.md

```diff
@@ -103,9 +116,9 @@ Select all that apply:
 ```
 [] `$(".post").css("background", "peachpuff")`
 [] `$(".post").innerHTML`
-[] `$(".post").html()`
-[] `document.getElementsByClassName("post")[0].innerHTML`
-[] `document.getElementsByClassName("post").innerHTML`
+[X] `$(".post").html()`
+[X] `document.getElementsByClassName("post")[0].innerHTML`
+[X] `document.getElementsByClassName("post").innerHTML`
 ```
```

Comment:
`$(".post").css("background", "peachpuff")` will work. We're using a jquery function on a jquery object.

`document.getElementsByClassName("post").innerHTML` will not work. `getElementsByClassName" always returns an array of DOM objects. We have to index it because we can't call `innerHTML` on an array.